argo_prod/solrconfig.xml

```diff
@@ -75,11 +145,11 @@
         contributor_text_nostem_im^3
         topic_tesim^2
 
-        exploded_project_tag_ssim^2
-        exploded_nonproject_tag_ssim
-        exploded_tag_ssim
         tag_ssim
 
+        originInfo_place_placeTerm_tesim
```

Comment:
from here below we are making the qttest requestHandler have the same qf, pf, pf3 and pf2 as the default requestHandler.  This will no doubt change over time.